pages/node/basic.md

```diff
@@ -49,6 +76,11 @@ Stop your currently running Axelar node:
 pkill -f "axelard start"
 ```
 
+Stop if you run node as service
+
+```bash
+sudo -S systemctl stop axelard
+```
```

Comment:
Can you add a simple instruction about deleting / unregistering the system service?